cypress/support/pages/signIn.pageObject.js

```diff
@@ -5,30 +5,34 @@ class SignInPageObject extends PageObject {
 
   get emailField() {
     return cy.getByDataCy('email-sign-in');
+    return this.getByDataCy('email-sign-in');
   }
 
   get passwordField() {
     return cy.getByDataCy('password-sign-in');
+    return this.getByDataCy('password-sign-in');
   }
 
   get signInBtn() {
     return cy.getByDataCy('sign-in-btn');
+    return this.getByDataCy('sign-in-btn');
   }
 
   typeEmail(email) {
     this.emailField
       .type(email);
+        .type(email);
   }
 
   typePassword(password) {
     this.passwordField
       .type(password);
+        .type(password);
   }
 
   clickSignInBtn() {
     this.signInBtn
       .click();
+        .click();
```

Comment:
There is a redundant click statement here. Remove the duplicate `.click();` to ensure the method is clear and concise.